spec/serializers/todo_serializer_spec.rb

```diff
@@ -1,9 +1,11 @@
 # frozen_string_literal: true
 
+include Rails.application.routes.url_helpers
+
 RSpec.describe TodoSerializer do
   subject { described_class.new(todo) }
 
-  let(:todo) { build(:todo) }
+  let(:todo) { create(:todo) }
```

Comment:
What's the difference between build and create?

Reply:
`build` constructs the active model, but it does not store it into the database. `create` constructs the active model and stores it into the database (aka assigning a value to ID field). I need `create` because I need the a value of ID to construct the URL, otherwise it would be nil with build.